Cargo.toml

```diff
@@ -683,36 +683,36 @@ visibility = "0.1.1"
-# alloy-serde = { git = "https://github.com/alloy-rs/alloy", rev = "cfb13aa" }
-# alloy-signer = { git = "https://github.com/alloy-rs/alloy", rev = "cfb13aa" }
-# alloy-signer-local = { git = "https://github.com/alloy-rs/alloy", rev = "cfb13aa" }
-# alloy-transport = { git = "https://github.com/alloy-rs/alloy", rev = "cfb13aa" }
-# alloy-transport-http = { git = "https://github.com/alloy-rs/alloy", rev = "cfb13aa" }
-# alloy-transport-ipc = { git = "https://github.com/alloy-rs/alloy", rev = "cfb13aa" }
-# alloy-transport-ws = { git = "https://github.com/alloy-rs/alloy", rev = "cfb13aa" }
-#
+[patch.crates-io]
+alloy-consensus = { git = "https://github.com/alloy-rs/alloy", rev = "20002c8" }
+alloy-contract = { git = "https://github.com/alloy-rs/alloy", rev = "20002c8" }
+alloy-eips = { git = "https://github.com/alloy-rs/alloy", rev = "20002c8" }
+alloy-genesis = { git = "https://github.com/alloy-rs/alloy", rev = "20002c8" }
+alloy-json-rpc = { git = "https://github.com/alloy-rs/alloy", rev = "20002c8" }
+alloy-network = { git = "https://github.com/alloy-rs/alloy", rev = "20002c8" }
+alloy-network-primitives = { git = "https://github.com/alloy-rs/alloy", rev = "20002c8" }
+alloy-node-bindings = { git = "https://github.com/alloy-rs/alloy", rev = "20002c8" }
+alloy-provider = { git = "https://github.com/alloy-rs/alloy", rev = "20002c8" }
+alloy-pubsub = { git = "https://github.com/alloy-rs/alloy", rev = "20002c8" }
+alloy-rpc-client = { git = "https://github.com/alloy-rs/alloy", rev = "20002c8" }
+alloy-rpc-types = { git = "https://github.com/alloy-rs/alloy", rev = "20002c8" }
+alloy-rpc-types-admin = { git = "https://github.com/alloy-rs/alloy", rev = "20002c8" }
+alloy-rpc-types-anvil = { git = "https://github.com/alloy-rs/alloy", rev = "20002c8" }
+alloy-rpc-types-beacon = { git = "https://github.com/alloy-rs/alloy", rev = "20002c8" }
+alloy-rpc-types-debug = { git = "https://github.com/alloy-rs/alloy", rev = "20002c8" }
+alloy-rpc-types-engine = { git = "https://github.com/alloy-rs/alloy", rev = "20002c8" }
+alloy-rpc-types-eth = { git = "https://github.com/alloy-rs/alloy", rev = "20002c8" }
+alloy-rpc-types-mev = { git = "https://github.com/alloy-rs/alloy", rev = "20002c8" }
+alloy-rpc-types-trace = { git = "https://github.com/alloy-rs/alloy", rev = "20002c8" }
+alloy-rpc-types-txpool = { git = "https://github.com/alloy-rs/alloy", rev = "20002c8" }
+alloy-serde = { git = "https://github.com/alloy-rs/alloy", rev = "20002c8" }
+alloy-signer = { git = "https://github.com/alloy-rs/alloy", rev = "20002c8" }
+alloy-signer-local = { git = "https://github.com/alloy-rs/alloy", rev = "20002c8" }
+alloy-transport = { git = "https://github.com/alloy-rs/alloy", rev = "20002c8" }
+alloy-transport-http = { git = "https://github.com/alloy-rs/alloy", rev = "20002c8" }
+alloy-transport-ipc = { git = "https://github.com/alloy-rs/alloy", rev = "20002c8" }
+alloy-transport-ws = { git = "https://github.com/alloy-rs/alloy", rev = "20002c8" }
```

Comment:
I'm guessing this is to get the new c-kzg version?

Reply:
this points to https://github.com/alloy-rs/alloy/pull/2273 which has all the logic for peerdas style sidecars

Reply:
Ah it links to: https://github.com/alloy-rs/alloy/pull/2273